src/transformers/models/llama/tokenization_llama.py

```diff
@@ -209,6 +226,8 @@ def _convert_id_to_token(self, index):
     def convert_tokens_to_string(self, tokens):
         """Converts a sequence of tokens (string) in a single string."""
         current_sub_tokens = []
+        # since we manually add the prefix space, we have to remove it
+        tokens[0] = tokens[0].strip(SPIECE_UNDERLINE)
```

Comment:
It would be a bit safer using `lstrip` here - as I believe it's always a prefix we're removing 
```suggestion
        tokens[0] = tokens[0].lstrip(SPIECE_UNDERLINE)
```

Reply:
Yes! Though there will never be a space on the left, still more readable thanks!